src/services/FungibleTokens.js

```diff
@@ -36,30 +46,77 @@ export default class FungibleTokens {
         this.account = account;
     }
 
+    getParsedTokenAmount(amount, symbol, decimals) {
+        const parsedTokenAmount = symbol === 'NEAR'
+        ? parseNearAmount(amount) 
+        : parseTokenAmount(amount, decimals);
+
+        return parsedTokenAmount;
+    }
+
+    getFormattedTokenAmount(amount, symbol, decimals) {
+        const formattedTokenAmount = symbol === 'NEAR'
+        ? formatNearAmount(amount, 5) 
+        : removeTrailingZeros(formatTokenAmount(amount, decimals, 5));
+
+        return formattedTokenAmount;
+    }
+
+    async getEstimatedTotalFees(contractName, accountId) {
+        const storageBalanceAvailable = await this.isStorageBalanceAvailable(contractName, accountId);
+        if (contractName && accountId && !storageBalanceAvailable) {
+            return new BN(FT_TRANSFER_GAS).add(new BN(FT_MINIMUM_STORAGE_BALANCE)).add(new BN(FT_STORAGE_DEPOSIT_GAS)).toString();
+        } else {
+            return FT_TRANSFER_GAS;
+        }
```

Comment:
:nit: chaining style + line length make this hard to read
```suggestion
        if (contractName && accountId && !storageBalanceAvailable) {
            return new BN(FT_TRANSFER_GAS)
            .add(new BN(FT_MINIMUM_STORAGE_BALANCE))
            .add(new BN(FT_STORAGE_DEPOSIT_GAS))
            .toString();
        } else {
            return FT_TRANSFER_GAS;
        }
```